ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -9,11 +9,9 @@ workflow:
     - chain: ipi-openstack-post
     env:
       OPENSTACK_EXTERNAL_NETWORK: "external"
-      OS_CLOUD: "openstack-cloud"
-      OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
+      OPENSTACK_INSTANCE_FLAVOR: "m1.s2.xlarge"
```

Comment:
standarized on the name "OPENSTACK_INSTANCE_FLAVOR" since we had two versions of the name and this one seems more appropriate.